core/domain/feedback_jobs_one_off_test.py

```diff
@@ -85,7 +85,7 @@ def test_message_count_job(self):
         self.assertEqual(thread2.message_count, None)
         self.assertEqual(thread3.message_count, None)
         job_output = self._run_one_off_job()
-        self.assertEqual(len(job_output[0]), 17)
+        self.assertEqual(job_output, [u"[u'SUCCESS', 3]"])#pylint: disable=invalid-string-quote
```

Comment:
Find a way to not use pylint-disable here.

Reply:
The output is in the double quotes itself, so to check I need to add double quotes in it (where pylint is giving error ```invalid-string-quote```).
Another way is to first strip the job_output (as per our requirement) then check it, but I think it is not good to do that.

Reply:
I think you could do `u'[u`'SUCCESS`', 3]']` or similar perhaps?

Reply:
As in, I don't see why there would be an issue if you replaced the outer double quotes with single quotes, as long as you correctly represent the interior single quotes..

Reply:
Hi @seanlip , thanks for the help !
DONE.